examples/resources/cyral_repository_access_rules/resource.tf

```diff
@@ -0,0 +1,36 @@
+resource "cyral_repository_access_rules" "some_resource_name" {
```

Comment:
I am facing an infinite `terraform apply` loop when trying out this example, where even though my .tf file hasn't changed, the Terraform plan kept suggesting this change:

```
Terraform will perform the following actions:

  # cyral_repository_access_rules.t1 will be updated in-place
  ~ resource "cyral_repository_access_rules" "t1" {
        # (2 unchanged attributes hidden)

      ~ rule {
            # (2 unchanged attributes hidden)

          + config {}

            # (1 unchanged block hidden)
        }
      ~ rule {
          + config {}

            # (1 unchanged block hidden)
        }

        # (1 unchanged block hidden)
    }
```

Don't know how to fix it out of the top of my head though :/

Reply:
Can you share with me the contents of your `.tf` file?

Reply:
I have pasted it below. It is essentially the configuration found in `examples/resources/cyral_repository_access_rules/resource.tf`.

```terraform
resource "cyral_repository" "t1" {
  name = "test1"
  type  = "sqlserver"
  port = 1433
  host = "sqlserver.local"
}

resource "cyral_repository_user_account" "t1" {
  repository_id = cyral_repository.t1.id
  name = "my-user-account"
  auth_scheme {
    aws_secrets_manager {
      secret_arn = "redacted"
    }
  }
}

resource "cyral_repository_access_rules" "t1" {
  repository_id = cyral_repository.t1.id
  user_account_id = cyral_repository_user_account.t1.id

  # First rule to be considered
  rule {
    identity {
      type = "username"
      name = "my-user-account"
    }
    config {
      policy_ids = []
    }
    valid_from = "2022-09-19T23:50:48.606Z"
    valid_until = "2022-10-10T00:00:00Z"
  }

  # Second rule to be considered
  rule {
    identity {
      type = "email"
      name = "fake-username@gmail.com"
    }
  }

  # ... and so on
  rule {
    identity {
      type = "group"
      name = "fake-group"
    }
    config {
      policy_ids = []
    }
  }
}
```